### PR TITLE
Drop `json_payload` column.

### LINKED ISSUE #890: api_review.json_payload table is being filled with non-json
Just took a look at the `api_review` table's `json_payload` column in the database and discovered it's currently being populated with `ApiDefinitionRequest.toString()` - which is definitely not JSON formatted. 

We do have a separate `api_review.api_definition` column with the typically YAML formatted body which makes me wonder whether the correct fix is to just drop the `json_payload` column entirely (or at least use of it).

### PR DESCRIPTION
**Pull request by vadeg**
Fixes #890

### CODE REVIEW

**Comment by vadeg** [comment]
👍

File: server/zally-server/src/main/resources/db/migration/V006__drop_json_payload_column.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE api_review DROP COLUMN json_payload;
```
Comment (tkrop) [inline server/zally-server/src/main/resources/db/migration/V006__drop_json_payload_column.sql:1]: Before we drop this, we should consequently look into our database, whether `api_definition` is really filled as expected for the entire review history and merge the content if necessary.

Comment (tkrop) [reply server/zally-server/src/main/resources/db/migration/V006__drop_json_payload_column.sql:1]: @vadeg I had a first look at the data. It looked actually, as if storing this data has been a dumb idea in the first place. I will approve as soon as the merge conflict is resolved.

**Comment by vadeg** [comment]
@tkrop the conflict has been resolved.

**Comment by vadeg** [comment]
👍

**Comment by vadeg** [comment]
@tkrop I have resolved conflicts. Please review again.

**Comment by tkrop** [comment]
:+1:

**Pull request merged**